### PR TITLE
chore: bump openapi

### PR DESCRIPTION
**Pull request by mfw78**
**Problem**
The services `openapi.yml` has drifted from the version published in the `cow-sdk`.

**Solution**
Let's update it!

### CODE REVIEW

File: src/order-book/generated/models/OrderPostError.ts [hunk 1]
```diff
@@ -10,27 +10,30 @@ export type OrderPostError = {
...
         TRANSFER_SIMULATION_FAILED = 'TransferSimulationFailed',
-        UNSUPPORTED_TOKEN = 'UnsupportedToken',
-        WRONG_OWNER = 'WrongOwner',
-        INVALID_EIP1271SIGNATURE = 'InvalidEip1271Signature',
-        MISSING_FROM = 'MissingFrom',
-        SAME_BUY_AND_SELL_TOKEN = 'SameBuyAndSellToken',
         ZERO_AMOUNT = 'ZeroAmount',
-        UNSUPPORTED_BUY_TOKEN_DESTINATION = 'UnsupportedBuyTokenDestination',
+        INCOMPATIBLE_SIGNING_SCHEME = 'IncompatibleSigningScheme',
+        TOO_MANY_LIMIT_ORDERS_UNSUPPORTED_BUY_TOKEN_DESTINATION = 'TooManyLimitOrders UnsupportedBuyTokenDestination',
```
Comment (anxolin) [inline src/order-book/generated/models/OrderPostError.ts:27]: u merged these two lines?

File: src/order-book/generated/models/OrderPostError.ts [hunk 2]
```diff
@@ -10,27 +10,30 @@ export type OrderPostError = {
...
         UNSUPPORTED_ORDER_TYPE = 'UnsupportedOrderType',
+        INSUFFICIENT_VALID_TO = 'InsufficientValidTo',
+        EXCESSIVE_VALID_TO = 'ExcessiveValidTo',
+        TRANSFER_ETH_TO_CONTRACT = 'TransferEthToContract',
+        INVALID_NATIVE_SELL_TOKEN_SAME_BUY_AND_SELL_TOKEN = 'InvalidNativeSellToken SameBuyAndSellToken',
         UNSUPPORTED_SIGNATURE = 'UnsupportedSignature',
-        TOO_MANY_LIMIT_ORDERS = 'TooManyLimitOrders',
-        INVALID_APP_DATA = 'InvalidAppData',
+        UNSUPPORTED_TOKEN = 'UnsupportedToken',
+        UNSUPPORTED_CUSTOM_INTERACTION_INVALID_APP_DATA = 'UnsupportedCustomInteraction InvalidAppData',
```
Comment (anxolin) [inline src/order-book/generated/models/OrderPostError.ts:36]: u merged two lines?

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cowprotocol/cow-sdk",
-  "version": "3.0.1",
+  "version": "3.0.2",
```
Comment (anxolin) [inline package.json:3]: mmm, will this be a breaking change for SDK clients?

File: src/order-book/generated/models/OrderPostError.ts (hunk 1 above)
Comment (mfw78) [reply src/order-book/generated/models/OrderPostError.ts]: These have just been generated straight from the `openapi.yml`.

File: package.json (hunk 3 above)
Comment (mfw78) [reply package.json:3]: Sure, bumped to major semver.

**Review by anxolin — APPROVED** [review]

**Pull request merged**